jest.config.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
 	testEnvironment: 'node',
 	resetMocks: true, // clear history and reset behavior of mocks between each test
 	restoreMocks: true, // restore initial behavior of mocked functions
+	testPathIgnorePatterns: ['/node_modules/', '/dist/'], // ignore dist folder
```

Comment:
I kept getting alerts from jest about collisions in the hastemap, so I added this to stop them.